.github/workflows/validate-pr-actions.yaml

```diff
@@ -2,34 +2,109 @@
 name: Validate Github Actions
 
 on:
-  workflow_call:
-    inputs:
-      image:
-        type: string
-        required: false
   pull_request:
     paths:
+      - Dockerfile
+      - src/**
+      - Pipfile
+      - Pipfile.lock
       - actions/**
       - mpyl_config.example.yml
       - tests/projects/job/**
       - tests/test_resources/changed-files/*
 
 # If multiple commits are pushed simultaneously, only build the last one.
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || inputs.image }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
   cancel-in-progress: false
 
 jobs:
+  changed-files:
+    name: 📋 Do we need to build?
+    runs-on: ubuntu-latest
+    outputs:
+      build-docker-image: ${{ steps.changed-files.outputs.any_changed == 'true' }}
```

Comment:
I guess this resolves to `'true'` since that's what you're comparing to on line 41, so why the expression at all?

Reply:
Technically this resolves to `true` (boolean) which is then converted to `'true'` [because job outputs are strings](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs#overview). 

If you mean "why `== 'true'`, then because copy/paste from the documentation 😛